src/core_ocean/shared/mpas_ocn_equation_of_state_jm.F

```diff
@@ -309,7 +309,7 @@ subroutine ocn_equation_of_state_jm_density(meshPool, scratchPool, nCells, k_dis
          enddo
       endif
 
-      !$omp do schedule(runtime) private(k, DRDT0, DKDT, DRHODT, DRDS0, DKDS, DRHODS)
+      !$omp do schedule(runtime)
```

Comment:
FYI, I merged this locally and got conflicts with the previous local merge #170. I resolved the conflicts by using the version from #170 but removing all omp private statements from @philipwjones's EOS update.